config/routes.rb

```diff
@@ -11,4 +16,8 @@
     resources :comments
     resources :likes
   end
+
+  get 'api/user/:id/posts' => 'users#user_posts_api', format: 'json'
+  get 'api/post/:id/comments' => 'posts#post_comments_api', format: 'json'
+  post 'api/post/:id/comments' => 'comments#post_create_api'
```

Comment:
- [ ] a Afin, When I tried to login it says that you are logged in but when I hit the API endpoint it says you need to sign in please try to fix this prob in order for your API app works fine without errors
<img width="384" alt="Screen Shot 2022-10-08 at 11 07 57" src="https://user-images.githubusercontent.com/61790587/194702436-f5f2cdb0-60a0-4916-81c4-fe0c5f0d50e5.png">
<img width="666" alt="Screen Shot 2022-10-08 at 11 08 26" src="https://user-images.githubusercontent.com/61790587/194702440-6f38461d-f3a0-45b3-b2de-b808aa84b2c2.png">